src/backend/tests/fixtures/get-test-server.ts

```diff
@@ -11,9 +11,10 @@ import ms from "ms";
 import builtNextJsFixture from "../../../../.nsm";
 import { getTestMinio } from "./get-test-minio";
 import superjson from "superjson";
+import { UserType } from "@/types/auth";
 
 export interface GetTestServerOptions {
-  authenticateAs?: "case_manager" | "para" | "admin";
+  authenticateAs?: UserType.CaseManager | UserType.Para | UserType.Admin;
```

Comment:
Do we want to just set the type to UserType, and make it default to UserType.User if it's not specified?

Reply:
^^ Just tried this, there's not much benefit because the seed file doesn't have a "UserType.User" key so it becomes more trouble than it's worth